src/libs/Url.js

```diff
@@ -18,7 +18,8 @@ function addTrailingForwardSlash(url) {
  */
 function getPathFromURL(url) {
     try {
-        const path = new URL(url).pathname;
+        const parsedUrl = new URL(url);
+        const path = parsedUrl.pathname + parsedUrl.search + parsedUrl.hash;
```

Comment:
There should be better way to get full path just removing host. Can you research on that?